SkeletonViewCore/Sources/Internal/Collections/CollectionSkeleton.swift

```diff
@@ -9,8 +9,8 @@
 import UIKit
 
 enum CollectionAssociatedKeys {
-    static var dummyDataSource = "dummyDataSource"
-    static var dummyDelegate = "dummyDelegate"
+    static var dummyDataSource: UInt8 = 0
+    static var dummyDelegate: UInt8 = 0
```

Comment:
I think it can works like this.

```suggestion
    static var dummyDataSource: StaticString = "dummyDataSource"
     static var dummyDelegate: StaticString = "dummyDelegate"
```